README.md

```diff
@@ -144,6 +188,28 @@ parallel (but not concurrent!)[^1] collection.
     collector simply means that the garbage collection workload can be
     parallelised across multiple worker threads.
 
+### Finalisation
+
+Finalisers are a common component of most tracing GCs which are used to run code
+for cleanup once an object dies (e.g.~closing a file handle or a database
+connection).
+
+Alloy takes a novel approach to finalisation compared to previous Rust GCs in
+that it uses existing drop methods as garbage collection finalisers, saving
+users the potentially error-prone task of manually writing both destructor and
+finaliser methods for GC managed objects.
+
+When a `Gc`'s underlying `GcBox`  becomes unreachable, Alloy will call its
```

Comment:
s/underlying GcBox/underlying contents/ ?

Reply:
Fixed in the rework of the finaliser section.